tracks/javascript/exercises/leap/mentoring.md

```diff
@@ -1,27 +1,67 @@
+The point of this exercise is to teach the student to use:
+- [Remainder (%)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder_()), also known as modulo
+- [Logical operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators)
+- [Operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)
+
 ### Reasonable solutions
 
 ```javascript
-export const isLeap = (year) => year % 4 === 0 && year % 100 !== 0 || year % 400 === 0;
+export const isLeap = (year) => year % 4 === 0 && (year % 100 !== 0 || year % 400 === 0);
 ```
+Variations check for truthy `year % m` and falsy `(!year % m)`.
 
 ```javascript
-export const isLeap = (year) => !(year % 4) && year % 100 || !(year % 400);
```

Comment:
The statement added in this pull request,

> Variations check for truthy `year % m` and falsy `(!year % m)`.

is less ambiguous without the removal of this snippet. Consider including both lines for greater clarity.

Reply:
Agreed. Added it back in!